### PR TITLE
Start of workshop

### PR DESCRIPTION
**Pull request by ingevandenende**
First solution file and presentation added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+2. Solution file for first part of workshop added
+3. Add solution file for second part of workshop
+
+To do:
+1. Improve based on feedback
+2. Create workshop notebooks after solution notebooks are final
 
 # An introduction to conformal prediction
-### Presentation: [Presentation_name](workshop/Presentation_template.pptx)
+### Presentation: [Conformal prediction](workshop/2024-01 PyLadies Amsterdam meet-up - Inge - Conformal prediction.pdf)
```
Comment (nancyirisarri) [inline README.md]: Hi, is this link working for you? I do not get it as a link.

**Review by nancyirisarri — CHANGES_REQUESTED** [review]
Hi,

Nice slides. I think they are very clear and well divided. The notebooks are also interesting. I have not checked them fully, but there seems to be missing notebooks in the _workshop_ folder. We usually separate unfinished or incomplete notebooks in the _workshop_ folder, and include the completed ones in the _solutions_ folder. During the session, the participants will work on completing the notebooks. Is this also your intention?

Also, it is a small detail, can you please try to commit the notebooks without the output of running the cells?

You could also consider providing instructions on adding the virtual environment as a kernel, something like these [https://janakiev.com/blog/jupyter-virtual-envs/](https://janakiev.com/blog/jupyter-virtual-envs/).

File: README.md (hunk 1 above)
Comment (ingevandenende) [reply README.md]: No, for me it is not working either. I just copied the name of the presentation into the template. How can I change it to a link?

**Comment by ingevandenende** [comment]
Thank you. Yes, my plan is to first finalize the solution file and after that create the workshop files to prevent double work. Do you have feedback on the code in the solution files?

I will update the notebooks to hide the output.

**Pull request merged**